components/org.wso2.carbon.appmgt.rest.api.publisher/src/main/java/org/wso2/carbon/appmgt/rest/api/publisher/impl/AdministrationApiServiceImpl.java

```diff
@@ -23,44 +23,230 @@
+            List<BusinessOwnerDTO> businessOwnerDTOList = new ArrayList<>();
+            List<BusinessOwner> businessOwners = apiProvider.getBusinessOwners();
+            List<BusinessOwnerProperty> businessOwnerPropertyList;
+            if (businessOwners.isEmpty()) {
+                return RestApiUtil.buildNotFoundException("Business Owners", null).getResponse();
+            }
+
+            for (BusinessOwner businessOwner : businessOwners) {
+                BusinessOwnerDTO businessOwnerDTO = new BusinessOwnerDTO();
+                businessOwnerDTO.setId(businessOwner.getBusinessOwnerId());
+                businessOwnerDTO.setName(businessOwner.getBusinessOwnerName());
+                businessOwnerDTO.setEmail(businessOwner.getBusinessOwnerEmail());
+                businessOwnerDTO.setDescription(businessOwner.getBusinessOwnerDescription());
+                businessOwnerDTO.setSite(businessOwner.getBusinessOwnerSite());
+                businessOwnerPropertyList = businessOwner.getBusinessOwnerPropertiesList();
+                List<BusinessOwnerPropertiesDTO> businessOwnerPropertiesDTOList = new ArrayList<>();
+                //save custom properties of the owner.
+                for (BusinessOwnerProperty businessOwnerProperty : businessOwnerPropertyList) {
+                    BusinessOwnerPropertiesDTO businessOwnerPropertiesDTO = new BusinessOwnerPropertiesDTO();
+                    businessOwnerPropertiesDTO.setKey(businessOwnerProperty.getPropertyId());
+                    businessOwnerPropertiesDTO.setValue(businessOwnerProperty.getPropertyValue());
+                    businessOwnerPropertiesDTO.setIsVisible(businessOwnerProperty.isShowingInStore());
+                    businessOwnerPropertiesDTOList.add(businessOwnerPropertiesDTO);
+                }
+                businessOwnerDTO.setProperties(businessOwnerPropertiesDTOList);
+                businessOwnerDTOList.add(businessOwnerDTO);
+            }
+            businessOwnerListDTO.setBusinessOwnerList(businessOwnerDTOList);
+        } catch (AppManagementException e) {
+            String errorMessage = "Error while retrieving business owners.";
+            RestApiUtil.handleInternalServerError(errorMessage, e, log);
+        }
+        return Response.ok().entity(businessOwnerListDTO).build();
     }
 
     @Override
     public Response administrationBusinessownerPost(BusinessOwnerDTO body, String contentType, String ifModifiedSince) {
-        return null;
+        beanValidator = new BeanValidator();
+        beanValidator.validate(body);
+        BusinessOwnerDTO businessOwnerDTO = new BusinessOwnerDTO();
+        try {
+            APIProvider apiProvider = RestApiUtil.getLoggedInUserProvider();
+            String ownerName = body.getName();
+            String ownerEmail = body.getEmail();
+            String ownerDescription = body.getDescription();
+            String ownerSite = body.getSite();
+            if (StringUtils.isEmpty(ownerName)) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/0d9719587a489bf2ef909c862c46bdbce01f1af0